src/pages/industries/financial-services.jsx

```diff
@@ -0,0 +1,230 @@
+  ],
+};
+
+const bulletSection2 = {
+  heading: 'Weave Compliance and Security Throughout the DevOps process.',
+  paragraphs: [
+    'Cilium empowers you to free your security and operations teams from the need to review each policy change manually. Cilium ensures that the appropriate TLS version and ciphers protect all traffic that needs to be encrypted, that the SNI matches the original destination DNS name, and that a trusted certificate authority signs the certificate received.',
+  ],
+  withImage: true,
+  imageSrc: FinanceImage2,
+  imageAlt: 'cilium TLS keys and certificate illustration',
+};
+
+const bulletSection3 = {
+  heading: '“Always On” Workload Analysis for Continuous Compliance',
+  paragraphs: [
+    'Cilium ensures continuous compliance with standards like FIPS, PCIDSS, Open Banking, Fedramp, and SOC 2 by offering real-time monitoring and analysis of workload communication to identify non-compliant connections. It emphasizes encryption, traffic analysis, and full-stack monitoring without imposing performance constraints, ensuring confidentiality, integrity, and availability in regulated environments.    ',
+  ],
+
+  withImage: true,
+  imageSrc: FinanceImage1,
+  imageAlt: 'Cilium continuous compliance image',
+};
+
+const financialUsecases = [
+  {
+    icon: EncryptionIcon,
+    title: 'Transparent Encryption ',
+    url: '/use-cases/transparent-encryption',
+    description:
+      'Elevate compliance and lower risk with Cilium transparent encryption. With just one switch, no application changes, service meshes  or additional proxies',
+  },
+  {
+    icon: NetworkPolicyIcon,
+    title: 'Network Policy ',
+    url: '/use-cases/network-policy',
+    description:
+      'Maintain identity based policies effectively at scale with Cilium’s advanced network polices ',
+  },
+  {
+    icon: EgressGatewayIcon,
+    title: 'Egress Gateway',
+    url: '/use-cases/egress-gateway',
+    description:
+      'Present a group of cloud native workloads from a stable IP address to integrate with traditional firewalls.',
+  },
+];
+
+const financialTalks = [
+  {
+    title: "eBPF, a road to invisible network: S&P Global's Network Transformation Journey",
+    description:
+      'Guru Ramamoorthy talks on how S&P Global’s network engineers leveraged eBPF-based networking with Cilium to power their application.',
+    videoSrc: 'https://www.youtube.com/embed/6CZ_SSTqb4g',
+  },
+  {
+    title: 'Cilium in Practice: Building Data Sandboxes at Bloomberg',
+    videoSrc: 'https://www.youtube.com/embed/8fiYVyISyz4',
+    description:
+      'Bloomberg leverages Cilium to construct data sandboxes that restrict users from distributing data outside the sandbox',
+  },
+  {
+    title: 'Multi-cluster networking with Cilium at Form3',
+    description: 'This address the reasons Form3 choosed to build its platform with Cilium',
```

Comment:
Can you come up with a better description here?

Reply:
Sure :)